flink-dist/pom.xml

```diff
@@ -108,6 +108,18 @@ under the License.
 			<version>${project.version}</version>
 		</dependency>
 
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-statebackend-changelog</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-dstl-dfs</artifactId>
+			<version>${project.version}</version>
+        </dependency>
```

Comment:
inconsistent indentation

Reply:
fixed in b14ababb8a5d1433710d69f2184b475c335dac2d